src/components/audioManagement/audioPlayer.ts

```diff
@@ -10,6 +10,7 @@ interface Player {
   getCurrentTime: () => number;
   getDuration: () => number;
   seek: (time: number) => void;
+  setSeconds: (seconds: number) => void;

```

Comment:
The seek() method is already doing the job of setSeconds. Furthermore it throws Peaks.js events such that the waveform is informed about visually changing the playhead.

Reply:
Ah, didn't notice that!